restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -111,12 +112,12 @@
 	"dialStepSipUriInputLabel": "Sip URI",
 	"dialStepSipUriInputPlaceholder": "sip:thomas@127.0.0.1:5080",
 	"dialStepSipUriInputTooltip": "Marque para SIP URI",
-	"dialStepContinueTooltip": "Seleccione módulo que continuará al finalizar la llamada. 'core_DialCallStatus', 'core_DialCallSid', 'core_DialCallDuration' y 'core_RecordingUrl' (si es permitido grabar) las variables estarán disponibles.",
+	"dialStepContinueTooltip": "Seleccione módulo que continuará al finalizar la llamada. Las siguientes variables estarán disponibles (si esta permitido grabar): 'core_DialCallStatus', 'core_DialCallSid', 'core_DialCallDuration' y 'core_RecordingUrl'.",

```

Comment:
Grammar error found. Change for:
"dialStepContinueTooltip": "Seleccione el módulo que continuará al finalizar la llamada. Las siguientes variables estarán disponibles (si está permitido grabar): 'core_DialCallStatus', 'core_DialCallSid', 'core_DialCallDuration' y 'core_RecordingUrl'.",